src/main/java/frc/robot/classes/TunableNumber.java

```diff
@@ -0,0 +1,38 @@
+package frc.robot.classes;
+
+import java.util.function.Consumer;
+import edu.wpi.first.networktables.GenericEntry;
+import edu.wpi.first.wpilibj.shuffleboard.Shuffleboard;
+import edu.wpi.first.wpilibj.shuffleboard.ShuffleboardTab;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.robot.Robot;
+
+public class TunableNumber extends SubsystemBase{
+    private ShuffleboardTab tuningTab = Shuffleboard.getTab("Tuning");
+    private GenericEntry numberEntry;
+    private String name;
+    private double value;
+    private Consumer<Double> consumer;
+
+    public TunableNumber(String name, double defaultValue, Consumer<Double> consumer){
+        this.name = name;
+        this.value = defaultValue;
+        this.consumer = consumer;
+        if (Robot.tuningMode){
```

Comment:
In addition to the constant, a shuffleboard entry to enable/disable tuning mode would be good to have. I think if you make it 'static' there will be one for the whole class

Reply:
We should talk a bit about what this looks like.

If tuningMode is ! isCompetition, then we wouldn't want a shuffleboard entry at all.

If tuningMode is separate from isCompetition, then it will need to be not final.  
And if isCompetition, we won't want another shuffleboard entry to enable/disable, as that involves a network call.

Reply:
I've created boolean tuningMode that can be toggled in shuffleboard. So if ! isCompetition and is tuningMode than it will proceed as usual.

Reply:
Do we need a tuning mode entry if we already have the constant? I'd rather we minimize on networktable entries when in competition mode